src/main/java/com/hms_networks/sc/cumulocity/config/CConnectorConfigFile.java

```diff
@@ -368,6 +381,25 @@ public boolean getCumulocitySubscribeToErrors() {
     return subscribeToErrors;
   }
 
+  /**
+   * Gets the Cumulocity port setting.
+   *
+   * @return Cumulocity port setting
+   */
+  public String getCumulocityPort() {
+    String port = PORT_DEFAULT;
+    try {
+      port =
+          configurationObject
+              .getJSONObject(CONFIG_FILE_CUMULOCITY_KEY)
+              .getString(CONFIG_FILE_PORT_KEY);
```

Comment:
I'm confused/concerned about this being a string. Does this mean the configuration file should like this? 
```JSON
{
    "Port": "1883"
}
```
I'm concerned that the following is what users will think, and the logs won't point out the error 
```JSON
{
  "Port": 1883
}
```
I would also be concerned about validating the port..  could you please test 
```JSON
{
  "Port": "shark"
}
```

Reply:
Will guard the value to ensure that it is a valid integer.

The Ewon MQTT port set method actually requires a string itself, thus the string data type is carried throughout here. Converting to an int and back for the configuration field would be a waste of computing time and introduces unneeded complexity.

Reply:
Changed - if the user includes a value which is not a valid number, the connector will catch and warn:

*Port does not contain a valid port number. The port must be a valid number between 0 and 65535, enclosed in a string. Using default: 8883*